src/Extensions/JsConsoleLogger/JsConsoleLogger.php

```diff
@@ -33,35 +34,81 @@ final class JsConsoleLogger extends AbstractWebDriverExtension
         Events::STEP_AFTER => 'afterStep',
     ];
 
+    /**
+     * Handle event
+     *
+     * @param StepEvent $event
+     * @return void
+     */
     public function afterStep(StepEvent $event): void
     {
         $this->checkJsErrors($event);
     }
 
+    /**
+     * Handle js errors if there are any
+     *
+     * @param StepEvent $event
+     * @return void
+     * @throws \Codeception\Exception\ModuleRequireException
+     */
```

Comment:
Same here please - Can you remove the superfluous @param and @return in the docblock they are already specified in the code. Thanks